projects/ancestor/iamright.py

```diff
@@ -0,0 +1,8 @@
+test = "hello"
+
+
+def amIright():
```

Comment:
Be careful about what you push to github. if the project doesn't need it. don't push it.